client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -136,7 +159,125 @@ const getActionsForMailbox = ( mailbox, translate, dispatch ) => {
 	};
 };
 
-function EmailPlanMailboxesList( { accountType, mailboxes, isLoadingEmails } ) {
+const getTitanMenuItems = ( translate ) => {
+	return [
+		{
+			href: 'https://wp.titan.email/mail/',
+			image: titanMailIcon,
+			imageAltText: translate( 'Titan Mail icon' ),
+			title: translate( 'Mail', {
+				comment: 'This refers to the Email application (i.e. the webmail) of Titan',
+			} ),
+		},
+		{
+			href: 'https://wp.titan.email/calendar/',
+			image: titanCalendarIcon,
+			imageAltText: translate( 'Titan Calendar icon' ),
+			title: translate( 'Calendar', {
+				comment: 'This refers to the Calendar application of Titan',
+			} ),
+		},
+		{
+			href: 'https://wp.titan.email/contacts/',
+			image: titanContactsIcon,
+			imageAltText: translate( 'Titan Contacts icon' ),
+			title: translate( 'Contacts', {
+				comment: 'This refers to the Contacts application of Titan',
+			} ),
+		},
+	];
+};
+
+const getGmailMenuItems = ( domainName, translate ) => {
+	return [
+		{
+			href: getGmailUrl( domainName ),
+			image: gmailIcon,
+			imageAltText: translate( 'Gmail icon' ),
+			title: translate( 'View Gmail' ),
+		},
+		{
+			href: getGoogleAdminUrl( domainName ),
+			image: googleAdminIcon,
+			imageAltText: translate( 'Google Admin icon' ),
+			title: translate( 'View Admin' ),
+		},
```

Comment:
We should not display that particular link for non-admin users, as they don't have access to the Google Admin Console. This can be addressed in another pull request though.